commands.js

```diff
@@ -35,14 +34,18 @@ function readFile(pth) {
 function addToFile(pth, newComm) {
   createIfNotExists(pth);
   fs.readFile(pth, (err, commands) => {
-    if (err) throw err;
+    if (err) {
+      console.log(err);
```

Comment:
For now, error handling is not ideal. Take a look at this scenario:
```bash
$ export SHORTC_PATH="/"
$ node shortc.js add --com "fail" --desc "fail"
fail fail
Error: EACCES: permission denied, open '/shortc.json'
    at Object.openSync (node:fs:583:3)
    at Object.writeFileSync (node:fs:2144:35)
    at createIfNotExists (/home/gdobn/Code/romka/shortc/commands.js:20:10)
    at addToFile (/home/gdobn/Code/romka/shortc/commands.js:35:3)
    at Object.addCommand (/home/gdobn/Code/romka/shortc/commands.js:59:3)
    at Object.handler (/home/gdobn/Code/romka/shortc/shortc.js:24:14)
    at /home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:8973
    at j (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:4852)
    at _.applyMiddlewareAndGetResult (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:8942)
    at _.runCommand (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:7127) {
  errno: -13,
  syscall: 'open',
  code: 'EACCES',
  path: '/shortc.json'
}
[Error: ENOENT: no such file or directory, open '/shortc.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/shortc.json'
}
undefined:1
undefined
^

SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at ReadFileContext.callback (/home/gdobn/Code/romka/shortc/commands.js:40:28)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:310:13)
```

Here I set `SHORTC_PATH` to `/`. Your program doesn't have rights to write to this location. So, your call to `createIfNotExists` didn't succeed:
```bash
Error: EACCES: permission denied, open '/shortc.json'
    at Object.openSync (node:fs:583:3)
    at Object.writeFileSync (node:fs:2144:35)
    at createIfNotExists (/home/gdobn/Code/romka/shortc/commands.js:20:10)
    at addToFile (/home/gdobn/Code/romka/shortc/commands.js:35:3)
    at Object.addCommand (/home/gdobn/Code/romka/shortc/commands.js:59:3)
    at Object.handler (/home/gdobn/Code/romka/shortc/shortc.js:24:14)
    at /home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:8973
    at j (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:4852)
    at _.applyMiddlewareAndGetResult (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:8942)
    at _.runCommand (/home/gdobn/Code/romka/shortc/node_modules/yargs/build/index.cjs:1:7127) {
  errno: -13,
  syscall: 'open',
  code: 'EACCES',
  path: '/shortc.json'
}
```

But, instead of terminating, the program continued to run, and then, the call to  `addToFile` failed:
```bash
[Error: ENOENT: no such file or directory, open '/shortc.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/shortc.json'
}
```

But again, instead of terminating, the program continued to run. And the call to `JSON.parse` didn't succeed, because we are passing `undefined` to it:
```bash
undefined:1
undefined
^

SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at ReadFileContext.callback (/home/gdobn/Code/romka/shortc/commands.js:40:28)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:310:13)
```

The best practice for error handling is the next: if we encounter the error, we should give to user as much information about the reason of error, and terminate the program. 
In your current design doing this would add some boilerplate. That's why I suggest you to rewrite your application with promises. This would allow us to handle errors in one place.

Othewise, we can define one erorr handling function. Something like this:
```js
const handleError = err => {
  console.error(err.message);
  processs.exit(1);
}
```
But this function gives information only about the error, but we want to also include where the error happens. And also, using `process.exit` [is not recommended](https://stackoverflow.com/questions/5266152/how-to-exit-in-node-js/37592669#37592669).

Reply:
To sum up, if you want, postpone error handling until better times. If you don't want, we can craft something with current situation.